view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -157,13 +157,13 @@ define(
 
                                 if (creditCardType in allInstallments) {
                                     // get for the creditcard the installments
-                                    var installmentCreditcard = allInstallments[creditCardType];
+                                    var cardInstallments = allInstallments[creditCardType];
```

Comment:
I know these variables aren't all related to this PR, but while we're at it, can we move away from `var` and have `let` or `const` instead?